reader.go

```diff
@@ -443,7 +474,6 @@ func decodeLineOfMasterPlaylist(p *MasterPlaylist, state *decodingState, line st
 				state.variant.HDCPLevel = v
 			}
 		}
-	case strings.HasPrefix(line, "#"):
 		// comments are ignored
```

Comment:
Nitpicking) Should we also remove this comment as this comment "justified" the removed case line 🤔